cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/AbstractContext.java

```diff
@@ -169,6 +172,27 @@
   protected final DynamicDatasetCache datasetCache;
   protected final RetryStrategy retryStrategy;
 
+
+  /**
+   * Returns runtime arguments as a Map.
+   * <ol>
+   *  <li>If INCLUDE_FEATURE_FLAGS_IN_RUNTIME_ARGUMENTS is set: extracting any feature flag defined in CConfiguration
```

Comment:
I think the feature name is different. I like this one more as it controls only runtime argument thing

Reply:
May be not al long, but something that includes `RUNTIME` would be good